src/collapsable-section/examples/collapsableSection.scss

```diff
@@ -1,6 +1,6 @@
 @import '../../utils/theme/variables.scss';
 
-.collapsable-section {
+.userinfo-collapsable-section {
```

Comment:
this way if the user changes the `className` in the example the style breaks.
why not doing `.live-demo .collapsable-section` instead?